### DISCUSSION TITLE
Brand new install - Tests just hang with no output

### DISCUSSION THREAD

**Discussion by benwrigley** [vitest-dev/vscode#301] [Q&A]
Before installing the plugin I was running tests fine with `npx vitest`. This automatically found my `vite.config.js`.

At first the plugin said there was no vitest config so I've copied my `vite.config.js` to `vitest.config.js` which now shows the tests in the testing window.

When I try to run the suite it just hangs with the test clock running. I'm not sure what I'm doing wrong.

Windows 10
VScode: 1.87.2
Vitest : 1.2.2

```
import { defineConfig } from 'vite';
import laravel from 'laravel-vite-plugin';
import vue from '@vitejs/plugin-vue';
import path from 'path';
import { run } from 'vite-plugin-run'

export default defineConfig({

  plugins: [
      laravel({
          input: 'resources/js/app.js',
          refresh: true,
      }),
      vue({
          template: {
              transformAssetUrls: {
                  base: null,
                  includeAbsolute: false,
              },
          },
      }),
      run({
          name: "generate translations",
          run: ['php', 'artisan', 'lingua:generate'],
          pattern: ['resources/lang/**', 'lang/**'],
      }),
  ],
  esbuild: {
    jsxFactory: 'h',
    jsxFragment: 'Fragment'
  },

  test: {
    globals: true,
    environment: 'happy-dom',
    include: ['resources/js/tests/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
  }
})
```

**Comment by sheremet-va** [comment]
Does pre-release version work for you? https://github.com/vitest-dev/vscode/releases/tag/v0.5.0

Note that it requires Vitest 1.4.0 or higher.

**Reply by benwrigley** [reply]
hi @sheremet-va, thanks for your fast reply!

I've moved to vitest 1.4.0 and pre-release and now they run on first load. I can see 32/35 passed in 2.3s

If I then try and 'Rerun last run' I get 0/0 in 39ms 

![image](https://github.com/vitest-dev/vscode/assets/18666603/9e9e86d7-ad95-41c8-bec5-92c94eb77544)

**Reply by sheremet-va** [reply]
I cannot reproduce it, sorry

**Reply by benwrigley** [reply]
No worries, I shall keep digging. I have a hunch it's a path issue somewhere.

If I click the `Run Tests` and select my config file from the drop down, they do successfully run again. But 'Rerun last run' or running any of the individual tests fails.

![image](https://github.com/vitest-dev/vscode/assets/18666603/a025bcc8-94a2-41ec-844c-8d9e6096a1f2)

**Reply by sheremet-va** [reply]
I just pushed a fix in 0.5.6. Please have a look when you have time. (It might take a few minutes to appear on the marketplace)

**Reply by benwrigley** [reply]
I'm afraid not no. Now I get a spinning wheel and 'Resolving Vitest'

![image](https://github.com/vitest-dev/vscode/assets/18666603/f9d9c4a2-578b-4a81-baef-e621b7431b12)

**Reply by sheremet-va** [reply]
Do you see anything in "Output" -> "Vitest"?

**Reply by benwrigley** [reply]
Sorry for slow reply:

```
[INFO 10:00:42 PM] [Vitest] Extension is activated because Vitest is installed or there is a Vite/Vitest config file in the workspace.
[INFO 10:00:44 PM] [API] Running Vitest: v1.4.0 (everyoneinmind), v1.4.0 (everyoneinmind), v1.4.0 (everyoneinmind), v1.4.0 (everyoneinmind)
```

**Reply by benwrigley** [reply]
In case this is helpful:

I have reinstalled version 0.5.4 (which is the only one that finds my test files)

The tests all run through once successfully  with lots of data in Output > Vitest

If I then select try to rerun (whether the whole suite or an invidivual test) I get nothing in Output but I see 'Error: Failed to parse source for import:

![image](https://github.com/vitest-dev/vscode/assets/18666603/27cc3b41-c145-410b-90b7-c7301baa4326)
